src/views/viewCommands.ts

```diff
@@ -808,6 +808,7 @@ export class ViewCommands {
 				pr.refs.head.branch,
 				repoIdentity.remote.url,
 				DeepLinkActionType.SwitchToPullRequestWorktree,
+				pr.id,
```

Comment:
We need to also pass PR ID in other places where we use "open in workspace" actions, like from the Launchpad and Launchpad view (`switchTo` of `launchpadProvider`): https://github.com/gitkraken/vscode-gitlens/blob/features/3549-open-multi-diff-editor-when-opening-a-new-worktree-from-a-pr/src/plus/launchpad/launchpadProvider.ts#L470-L471

This explains why I was not able to see the PR changes when I was testing your PR (I was using Launchpad).